src/query/functions-v2/src/scalars/variant.rs

```diff
@@ -127,15 +124,11 @@ pub fn register(registry: &mut FunctionRegistry) {
         |_| None,
         vectorize_with_builder_1_arg::<VariantType, NullableType<VariantType>>(|v, output, _| {
             if v.is_empty() {
-                output.validity.push(false);
-                output.builder.commit_row();
+                output.push_null()
             } else {
                 match object_keys(v) {
-                    Some(keys) => output.push(Some(&keys)),
-                    None => {
-                        output.validity.push(false);
-                        output.builder.commit_row();
```

Comment:
We don't need to `commit_row` in this case?

Reply:
`push_null` call `Variant::push_default`, which will call `commit_row`.